### DISCUSSION TITLE
Welcome to Termux-Hydra Discussions!

### DISCUSSION THREAD

**Discussion by iamunixtz** [iamunixtz/Termux-Hydra#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yamrajjjii** [comment]
I am unable to update password?

**Reply by iamunixtz** [reply]
U need to add ur own password files and usernames or u can use our password list to add ur own password.txt file and username.txt file u can use command 
```
termux-setup-storage
cd downloads
cp yourfile.txt $HOME /Termux-Hydra
```

**Comment by hackerdechezwich** [comment]
Good evening, I have a problem. When I enter my IP address, it asks me for a username and password, and I don't know how to do it. Can you tell me or give me a username or password Please ?
